credentialz/credentialz.proto

```diff
@@ -435,6 +435,9 @@ enum KeyType {
 
 message AccountCredentials {
   message AuthorizedKey {
+    // AuthorizedKey is an ssh key/idenity. The basic openssh format follows:
+    // ecdsa-sha2-nistp521 AAAkeystuffhere chuck@chuckfinley.org
+    //   key_type         authorized_key    description
     bytes authorized_key = 1;
```

Comment:
The format of this field (`authorized_key`) should be explicitly indicated, as Mel noted above, I think the above comment refers to the message instead

Reply:
done, moved the comment up for mel's comment and clarified that the key-type is mandatory.